source/OptimaJet.DWKit.StarterApplication/Services/Workflow/WorkflowProductService.cs

```diff
@@ -141,21 +137,36 @@ public async Task ProductProcessChangedAsync(ProductProcessChangedArgs args)
 
             await RemoveTasksByProductId(product.Id);
 
+            var tasks = await ReassignUserTasksForProduct(product);
```

Comment:
`ReassignUserTasksForProduct` calls `GetCurrentTaskCommnt` and then `RemoveTasksByProductId`.  It looks lik the call to `RemoveTasksByProductId` above this should be removed.

Reply:
oh good catch!